### ISSUE TITLE
PR preview is broken in VS 2019 16.7.3 on Windows 7

### ISSUE THREAD

**Issue by bchavez** [github/VisualStudio#2547]
## Versions
 - GitHub Extension for Visual Studio version:  
    ![image](https://user-images.githubusercontent.com/478118/93020510-5814a600-f592-11ea-8d92-b22fce582726.png)

 - Visual Studio version:
    ![image](https://user-images.githubusercontent.com/478118/93020525-6d89d000-f592-11ea-90a2-2ca06ea240fe.png)

## What happened
Clicking on a PR results in the following UI error:
![image](https://user-images.githubusercontent.com/478118/93020565-a45fe600-f592-11ea-9f18-9975dcf072cb.png)
![image](https://user-images.githubusercontent.com/478118/93020568-aa55c700-f592-11ea-93e4-d9b68b469eda.png)

Then, pressing Refresh:
![image](https://user-images.githubusercontent.com/478118/93020573-b2156b80-f592-11ea-854a-78baf79323ca.png)
![image](https://user-images.githubusercontent.com/478118/93020579-bcd00080-f592-11ea-8ff8-ca0fa8703d7e.png)

However, starting Visual Studio with `devenv /log` actually makes the GitHub window work:

![image](https://user-images.githubusercontent.com/478118/93021143-1423a000-f596-11ea-9e8a-e1916cbb0de4.png)
![image](https://user-images.githubusercontent.com/478118/93021150-1e459e80-f596-11ea-8a2f-99d8a88cbc92.png)

### Steps to Reproduce
1. Checkout https://github.com/bchavez/Coinbase.Pro on Windows 7.
1. Start VS without `/log` argument.
1. Open the Coinbase.Pro sln file.
1. Open the GitHub VS extension window.
1. Try checking out any PR on Windows 7 with VS 2019 16.7.3 installed
1. See error

Starting Visual Studio with `devenv /log` will make the error go away.

### Expected behavior
The expected behavior is that no error is shown and PR preview page is visible.
![image](https://user-images.githubusercontent.com/478118/93021159-21408f00-f596-11ea-8b35-fec6345eac61.png)

### Screenshots
See above.

#### GitHubVisualStudio\extension.log
```
2020-09-13 07:15:48.191 [18672] EROR [12] TaskExtensions            
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in d:\a\1\s\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in d:\a\1\s\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in d:\a\1\s\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in d:\a\1\s\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123<---

2020-09-13 07:16:04.851 [18672] INFO [129] GitHubPackage             Initializing GitHub Extension v2.11.102.28613 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:20:45.444 [20508] EROR [32] TaskExtensions            
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123<---

2020-09-13 07:20:56.368 [20508] INFO [131] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:23:52.275 [20508] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:23:55.141 [20508] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:23:55.801 [20508] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:23:56.014 [20508] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:23:56.185 [20508] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:25:08.398 [11664] EROR [56] TaskExtensions            
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123<---

2020-09-13 07:25:13.087 [11664] INFO [125] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:28:29.103 [11664] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:39:49.140 [12984] INFO [130] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:42:25.425 [18896] EROR [48] TaskExtensions            
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123<---

2020-09-13 07:42:45.660 [18896] INFO [23] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:45:01.435 [02668] INFO [95] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:47:19.365 [15216] EROR [29] TaskExtensions            
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Octokit.GraphQL.Model.UserConnection Octokit.GraphQL.Model.Repository.AssignableUsers(System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<Int32>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>, System.Nullable`1<Octokit.GraphQL.Core.Arg`1<System.String>>)'.
   at GitHub.Services.PullRequestService.<ReadAssignableUsers>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)
   at GitHub.Services.PullRequestService.ReadAssignableUsers(HostAddress address, String owner, String name, String after)
   at GitHub.ViewModels.GitHubPane.PullRequestListViewModel.LoadAuthors(String after) in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestListViewModel.cs:line 95
   at GitHub.ViewModels.UserFilterViewModel.<Load>d__23.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\UserFilterViewModel.cs:line 123<---

2020-09-13 07:47:22.626 [15216] INFO [92] GitHubPackage             Initializing GitHub Extension v2.11.104.25275 in Microsoft Visual Studio 2019 (16.7.30503.244)
2020-09-13 07:48:00.796 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:01.866 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:02.132 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:02.318 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:08.653 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:08.842 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:09.038 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0
2020-09-13 07:48:09.224 [15216] EROR [01] PullRequestDetailViewModel Error loading pull request bchavez/Coinbase.Pro/19 from GitHub
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub.ViewModels.GitHubPane.PullRequestDetailViewModel.<Refresh>d__118.MoveNext() in D:\a\VisualStudio\VisualStudio\src\GitHub.App\ViewModels\GitHubPane\PullRequestDetailViewModel.cs:line 0

```

#### `dotnet --info`
```
C:\>dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.1.402
 Commit:    9b5de826fd

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.402\

Host (useful for support):
  Version: 3.1.8
  Commit:  9c1330dedd

.NET Core SDKs installed:
  2.1.518 [C:\Program Files\dotnet\sdk]
  2.1.810 [C:\Program Files\dotnet\sdk]
  2.2.110 [C:\Program Files\dotnet\sdk]
  2.2.207 [C:\Program Files\dotnet\sdk]
  3.1.402 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.19 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.22 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.19 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.22 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.19 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.21 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.22 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by bchavez** [comment]
Hm. Seems like deleting the files in the `ComponentModelCache` may have helped:
```
C:\Users\USER\AppData\Local\Microsoft\VisualStudio\16.0_2a8eeedb\ComponentModelCache
```
Feel free to close the issue if you don't think it's an issue or this issue doesn't need extra error handling.

**Comment by bchavez** [comment]
Ugh. Nevermind, the bug is back:
![image](https://user-images.githubusercontent.com/478118/93022632-26560c00-f59f-11ea-91f3-5747a01354d5.png)

**Comment by jcansdale** [comment]
Hi @bchavez,

Thanks for the report!

I have a possible fix ready. Could you try installing the version from here:
https://github.com/github/VisualStudio/actions/runs/253994902

If this works, I'll get a new version out ASAP!

**Comment by bchavez** [comment]
Ok, back on my machine. Just some more data points:

I couldn't reproduce the issue from other PRs until I submitted a PR that has a failing status check: https://github.com/bchavez/Coinbase.Pro/pull/22

Now I can get the GitHub window to display the error *somewhat* reliably (about 8 in 10 times):

![image](https://user-images.githubusercontent.com/478118/93149249-b16ff880-f6aa-11ea-95f2-aaabe1e658dd.png)
![image](https://user-images.githubusercontent.com/478118/93149369-f85dee00-f6aa-11ea-8b01-3f4e22cc4504.png)

I will install the custom build now.

**Comment by bchavez** [comment]
Hi @jcansdale, I'm happy to report I started VS 2019 almost 25 times and I no longer get any error with the custom build! I think you have fixed the problem! Your new build is working great!

Thank you so much! :partying_face:

**Comment by jcansdale** [comment]
> I couldn't reproduce the issue from other PRs until I submitted a PR that has a failing status check: bchavez/Coinbase.Pro#22

Interesting, that must be why we're not getting a flood for reports.

> I'm happy to report I started VS 2019 almost 25 times and I no longer get any error with the custom build! I think you have fixed the problem! Your new build is working great!

@bchavez excellent, thanks for testing it!

**Comment by rosrosros** [comment]
Installed version 2.11.108 from here: https://github.com/github/VisualStudio/actions/runs/364077069
This solved the issue with VS version 16.8.1

**Comment by tannenbaums** [comment]
I just got the latest version 2.11.102.28613 on a brand new vs install and still getting the error above.
